runtime/utils.go

```diff
@@ -64,11 +64,14 @@ func PreInitMaster0(sshClient ssh.Interface, remoteHostIP string) error {
 	logger.Info("send cluster file to %s success !", remoteHostIP)
 
 	// send register login info
-	err = sshClient.Copy(remoteHostIP, common.DefaultRegistryAuthConfigDir(), common.DefaultRegistryAuthConfigDir())
-	if err != nil {
-		return fmt.Errorf("send register config to remote host %s failed:%v", remoteHostIP, err)
+	authFile := common.DefaultRegistryAuthConfigDir()
+	if utils.IsFileExist(authFile) {
+		err = sshClient.Copy(remoteHostIP, authFile, authFile)
+		if err != nil {
+			return fmt.Errorf("failed to send register config %s to remote host %s err: %v", authFile, remoteHostIP, err)
+		}
+		logger.Info("send register info to %s success !", remoteHostIP)
```

Comment:
we should ensure whether authFile is necessary, and if we should warn users when the file does not exist